core/src/main/java/com/github/steanky/element/core/factory/BasicCollectionCreator.java

```diff
@@ -8,17 +8,16 @@
 import java.util.*;
 import java.util.function.Function;
 
-import static com.github.steanky.element.core.util.Validate.*;
+import static com.github.steanky.element.core.util.Validate.elementException;
 
 /**
  * Basic implementation of {@link CollectionCreator}.
  */
 public class BasicCollectionCreator implements CollectionCreator {
     private static final Function<? super Class<?>, ? extends Class<?>> DEFAULT_RESOLVER = type -> {
-        if(type.equals(List.class) || type.equals(Collection.class)) {
+        if (type.equals(List.class) || type.equals(Collection.class)) {
             return ArrayList.class;
-        }
-        else if (type.equals(Set.class)) {
+        } else if (type.equals(Set.class)) {
             return HashSet.class;
         }
```

Comment:
maybe I should stop commenting on this

Reply:
yes, it's using a different editorconfig